typeset/converter.js

```diff
@@ -38,14 +38,24 @@ const createMapOfMathMLElements = async (log, inputPath, cssPath, outputPath, ou
   })
   const page = await browser.newPage()
 
+  // Disable resources, uses too much memory
+  await page.setRequestInterception(true)
+  page.on('request', (interceptedRequest) => {
+    const url = interceptedRequest.url().split('#')[0].split('?')[0]
+    if (url.startsWith('file:') && (url.endsWith('.xhtml') || url.endsWith('.html') || url.endsWith('.css'))) {
```

Comment:
In which case we need `.xhtml`, `.html` files to be imported?

Reply:
That's actually the input file.  If I don't include `.xhtml` (I don't know if the input file can be `.html` :thinking: :question:) then the input file isn't loaded and we can't do anything :sweat_smile:

Reply:
Ahh, tricky :D I thought we are launching puppeteer from file, but it was JSDOM functionality.